Parsers/News Article Parser.js

```diff
@@ -0,0 +1,37 @@
+/*
+activation_example:!parse_news name of news website
+regex:!parse_news
+flags:gmi
+*/
+
+var where = current.text.indexOf('!parse_news ') + 12;
+var term = current.text.substr(where).trim();
```

Comment:
I'd just change this to a `.replace("!news ", "")` and update the regex at the top to `!news`

Reply:
@chelming This has been addressed in the new commit